app/models/ems_folder.rb

```diff
@@ -177,7 +162,7 @@ def folder_path_objs(*args)
     options = args.extract_options!
     folders = path(:of_type => "EmsFolder")
     folders = folders[1..-1] if options[:exclude_root_folder]
-    folders = folders.reject { |f| NON_DISPLAY_FOLDERS.include?(f.name) } if options[:exclude_non_display_folders]
+    folders = folders.reject(&:hidden?) if options[:exclude_non_display_folders]
```

Comment:
Can set `:default => false` when adding the column in the migration, do you know if that will update existing rows or do we need an explicit data migration?